renderer/components/Request/Request.js

```diff
@@ -46,6 +48,17 @@ class Request extends React.Component {
     }
   }
 
+  /**
+   * getMaxOneTimeReceive - User's max one time receive capacity in satoshis.
+   *
+   * @returns {number} user's max one time receive capacity in satoshis
+   * @memberof Request
+   */
+  getMaxOneTimeReceive() {
+    const { cryptoUnit, maxOneTimeReceive } = this.props
+    return convert('sats', cryptoUnit, maxOneTimeReceive)
```

Comment:
Comment says that it returns amount in satoshi, but this looks to be returning in `cryptoUnit`

Reply:
updated